src/patient-registration/patient-registration.test.tsx

```diff
@@ -1,19 +1,43 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { render, fireEvent, wait } from '@testing-library/react';
+import { render, fireEvent, wait, getByLabelText, cleanup } from '@testing-library/react';
 import dayjs from 'dayjs';
 import { PatientRegistration } from './patient-registration.component';
+import { useConfig } from '@openmrs/esm-module-config';
+import { ConfigMock } from '../../__mocks__/openmrs-esm-config.mock';
+//import { ConfigMock } from '../../__mocks__/openmrs-esm-module-config.mock';
```

Comment:
@brandones @nickjhill14 For some reason if i use the commented out line to fetch the mock config the tests fail....Could it be an issue of how am exporting the constant otherwise the content is similar to those in the file imported on line 7

Reply:
That is very strange. What's the error?

Reply:
https://pastebin.com/KR0GTdNr

Reply:
Oh right, so, Jest uses that mock file to mock out the actual import. So maybe try
```
import { ConfigMock } from '@openmrs/esm-module-config'
```
?